style.css

```diff
@@ -0,0 +1,8 @@
+.title {
+  font-family: PT Serif;
+  font-weight: 700;
+}
+.subtitle {
+  font-family: Roboto;
+  font-weight: 500;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6280509)